### PR TITLE
issue HELM-2

### PR DESCRIPTION
**Pull request by veronikaKochugova**
scenario support added

### CODE REVIEW

File: mongoose/templates/scenario-configmap.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Release.Name }}-scenario-cm
+data:
+  scenario.js: |
+    {{ .Values.scenario }}
```
Comment (akurilov) [inline mongoose/templates/scenario-configmap.yaml]: Missing newline

**Review by akurilov — CHANGES_REQUESTED** [review]

File: mongoose/templates/scenario-configmap.yaml (hunk 1 above)
Comment (akurilov) [reply mongoose/templates/scenario-configmap.yaml]: https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

**Review by akurilov — APPROVED** [review]

**Comment by akurilov** [comment]
Need the readme note about how to use a custom scneario

**Review by akurilov — CHANGES_REQUESTED** [review]
Need the readme note about how to use a custom scneario

**Review by akurilov — APPROVED** [review]

**Review by dadlex — APPROVED** [review]

**Pull request merged**